### ISSUE TITLE
Display Row number in column "No"

### ISSUE THREAD

**Issue by CHOMNANP** [Laravel-Backpack/CRUD#1096]
I would like to display sequence of "row" like 1, 2, 3, 4, 5 , 6 in column "No" in the table.
Any idea how I can achieve this?

Thanks,
Punleu

**Comment by tabacitu** [comment]
Hi @CHOMNANP ,

You should be able to achieve this by modifying the Datatables JS in ```list.blade.php```. Datatables has an easy way to achieve this, it's called an index column: https://datatables.net/examples/api/counter_columns.html

Hope it helps. Cheers!

**Comment by promatik** [comment]
@tabacitu I needed this, so I did it, but I had to add the index column on the server side, since the table is populated by ajax, and I couldn't find a way to override the ajax response on JS DataTables.

Do you think this could be a new feature for backpack?
If so I can make a commit and push request.

Maybe @lloy0076 may also want to give his feedback.

**Comment by tabacitu** [comment]
@promatik sure, it sounds like a good feature. I suggest you open another issue with details about your implementation, to understand it, and only after that submit the PR. Thank you, cheers!
